pageserver/src/tenant/timeline.rs

```diff
@@ -2785,12 +2785,12 @@ impl Timeline {
     }
 
     /// Flush one frozen in-memory layer to disk, as a new delta layer.
-    #[instrument(skip_all, fields(tenant_id=%self.tenant_shard_id.tenant_id, shard_id = %self.tenant_shard_id.shard_slug(), timeline_id=%self.timeline_id, layer=%frozen_layer))]
```

Comment:
This was a mistake to remove the layer=%frozen_layer.

Reply:
I'm adding this to an upcoming PR